### DISCUSSION TITLE
How to access the getting started modules

### DISCUSSION THREAD

**Discussion by cherylbearden** [oqtane/oqtane.framework#858] [Q&A] [answered]
Hi, I downloaded and installed the oqtane framework and would like to get started building the sample modules.  When I tried to start with the 'Hello World', module, it took me back to this url: 
http://lightswitchhelpwebsite.com/Blog.aspx and I think the url that I want is :
http://lightswitchhelpwebsite.com/Blog/tabid/61/EntryId/4330/Creating-a-Hello-World-Module-For-Blazor-Oqtane.aspx
which I was unable to access.  Any ideas why?  Thanks very much!

**Comment by dtquocbao** [comment, marked as answer]
I guess you can try with the tutorial videos on Youtube. Search for "oqtane" then you will see the complete bundle of videos with cover most of basic things with Oqtane. You also can give your questions here so folks can help. The creator and his team also very proactive in support.

Good luck

**Comment by sbwalker** [comment]
@ADefWebserver migrated his legacy LightSwitchHelpWebsite to Blazor and is used a new Url: https://blazorhelpwebsite.com - so perhaps some of the older links were broken. I would suggest you read the Module Creator article: https://blazorhelpwebsite.com/ViewBlogPost/4 or, as @rockydant mentioned, the video series is also a good place to  start: https://www.youtube.com/watch?v=JPfUZPlRRCE&list=PLYhXmd7yV0elLNLfQwZBUlM7ZSMYPTZ_f

**Reply by cherylbearden** [reply]
Thank Shaun! I will check out the article and video. Very exciting development platform!

**Comment by ADefWebserver** [comment]
@cherylbearden Please let me know what pages have broken links. If they are on LightSwitchHelpWebsite.com or BlazorHelpWebsite.com I can fix them. Thank You

**Reply by cherylbearden** [reply]
The url that doesn't work for me is: http://lightswitchhelpwebsite.com/Blog/tabid/61/EntryId/4330/Creating-a-Hello-World-Module-For-Blazor-Oqtane.aspx.  It takes me instead to http://lightswitchhelpwebsite.com/Blog.aspx. Thanks!

**Reply by ADefWebserver** [reply]
> The url that doesn't work for me is: http://lightswitchhelpwebsite.com/Blog/tabid/61/EntryId/4330/Creating-a-Hello-World-Module-For-Blazor-Oqtane.aspx.

@cherylbearden - What page is that link on? Thanks!

**Reply by cherylbearden** [reply]
The page is: https://www.oqtane.org/Showcase. Then this page is displayed instead: http://lightswitchhelpwebsite.com/Blog.aspx.  And I see this in red if I inspect: discussions:1 Unchecked runtime.lastError: The message port closed before a response was received.
Thanks!

**Reply by ADefWebserver** [reply]
> The page is: https://www.oqtane.org/Showcase. Then this page is displayed instead: http://lightswitchhelpwebsite.com/Blog.aspx. And I see this in red if I inspect: discussions:1 Unchecked runtime.lastError: The message port closed before a response was received.
> Thanks!

@sbwalker Can you delete all the links I have in the Showcase? I tried but it won't let me. Thanks!

**Reply by sbwalker** [reply]
@ADefWebserver I renamed the page to Docs and pointed to the docs site: https://docs.oqtane.org/

**Reply by ADefWebserver** [reply]
> I renamed the page to Docs and pointed to the docs site

@sbwalker - Perfect. I will update that site with the links properly organized later this week.
